docs/develop/Code/environment.md

```diff
@@ -41,7 +41,7 @@ cd /path/to/taxonworks/
 ```
 If using [Docker environment](https://github.com/SpeciesFileGroup/install_taxonworks/blob/master/development/docker/README.md) then continue with:
 ```
-docker-compose exec app bash
+docker compose exec app bash
 ```
 That puts you in a bash shell inside the linux Docker image.
```

Comment:
```suggestion
That puts you in a bash shell inside the linux Docker image.
Older installations may require `docker-compose` instead of `docker compose`.
```